src/TurbulenceClosures/turbulence_closure_implementations/leith_enstrophy_diffusivity.jl

```diff
@@ -135,8 +135,8 @@ end
 
     νₑⁱʲᵏ = ℑxᶠᵃᵃ(i, j, k, grid, νₑ)
 
-    ∂x_c = ∂xᶠᶜᶜ(i, j, k, grid, c)
-    ∂z_c = ℑxzᶠᵃᶜ(i, j, k, grid, ∂zᶜᶜᶠ, c)
+    ∂x_c = ∂xᶠᶜᶜ(i, j, k, grid, tracer)
+    ∂z_c = ℑxzᶠᵃᶜ(i, j, k, grid, ∂zᶜᶜᶠ, tracer)
```

Comment:
I prefer `c`. It's easy to confuse `tracer` and `tracers`.

Reply:
That's a good point. I was mainly going for consistency with `tracers`. I'm going to change this one back.

Reply:
Rolled back. I kept the `tracer` instances in the AMD code though because it appears there in instances of the "`for tracer in tracers`" type, so it's clearer and there's nothing called `∂z_c`.